handle/UnitGroup.lua

```diff
@@ -0,0 +1,21 @@
+require('CeresStdLib.base.Native')
+
+local __u

```

Comment:
I'm not sure if using one local is the best solution now, since we're in Lua.
Exactly, you can modify local environment of the called function, where you then define `GetEnumUnit()` referencing your current unit.